__init__.py

```diff
@@ -2396,11 +2382,15 @@ def register():
     for w in bpy.context.window_manager.windows:
         for a in w.screen.areas:
             if a.type == "PREFERENCES":
-                tasks_queue.add_task(
-                    (bpy.ops.wm.blenderkit_welcome, ("INVOKE_DEFAULT",)),
-                    fake_context=True,
-                    fake_context_area="PREFERENCES",
-                )
+                # DISABLED because causes crashes in Blender 4.1
```

Comment:
@vilemduha In Blender 4.1 the Welcome window fails to open (also in pre-client add-on) with error:

```
Read prefs: "/Users/ag/Library/Application Support/Blender/4.1/config/userpref.blend"
blenderkit addon: Verbose is enabled
blenderkit addon: Wrote out updater JSON settings with content:
blenderkit addon: {'last_check': '', 'backup_date': '', 'update_ready': False, 'ignore': False, 'just_restored': False, 'just_updated': False, 'version_text': {}}
blenderkit updater: Running background check for update
blenderkit addon: Starting background checking thread
blenderkit addon: Checking for update now in background
blenderkit addon: Checking for update function
blenderkit INFO: Global dir permissions are OK. [15:40:42.542, utils.py:1281]
blenderkit addon: Read in JSON settings from file
blenderkit addon: Getting tags from server
Writing userprefs: "/Users/ag/Library/Application Support/Blender/4.1/config/userpref.blend" ok
Info: Preferences saved
blenderkit INFO: Skipping dependencies copy: bundled for python 3.10, running on python 3.11.7 (main, Feb  6 2024, 17:57:25) [Clang 15.0.0 (clang-1500.1.0.2.5)] [15:40:42.543, dependencies.py:23]
blenderkit INFO: Dependencies are available [15:40:42.621, dependencies.py:111]
blenderkit ERROR: task queue failed task:<function bpy.ops.wm.blenderkit_welcome at 0x2d4b98a60'>('INVOKE_DEFAULT',)Overriding context with temporary screen isn't supported [15:40:42.632, tasks_queue.py:142]
Writing userprefs: "/Users/ag/Library/Application Support/Blender/4.1/config/userpref.blend" ok
Info: Preferences saved
blenderkit INFO: Daemon is running on port 62485! [15:40:42.636, timer.py:87]
Traceback (most recent call last):
  File "/Users/ag/Library/Application Support/Blender/4.1/scripts/addons/blenderkit/timer.py", line 114, in daemon_communication_timer
    handle_task(task)
  File "/Users/ag/Library/Application Support/Blender/4.1/scripts/addons/blenderkit/timer.py", line 243, in handle_task
    return disclaimer_op.handle_disclaimer_task(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ag/Library/Application Support/Blender/4.1/scripts/addons/blenderkit/disclaimer_op.py", line 258, in handle_disclaimer_task
    disclaimer = task.result["results"][0]
                 ~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
Writing: /var/folders/n5/zgtk48652gq_1_b8h9g2mfph0000gn/T/blender.crash.txt
[1]    24912 segmentation fault  /Applications/Blender.app/Contents/MacOS/blender
```

Similiar for Client:

```
blenderkit addon: Running background update callback
blenderkit addon: BG thread: Finished check update, no callback
blenderkit ERROR: task queue failed task:<function bpy.ops.wm.blenderkit_welcome at 0x17e08f850'>('INVOKE_DEFAULT',)Overriding context with temporary screen isn't supported [15:05:40.744, tasks_queue.py:142]
Writing userprefs: "/Users/ag/Library/Application Support/Blender/4.1/config/userpref.blend" ok
Info: Preferences saved
blenderkit WARNING: Request for daemon reports failed: <class 'requests.exceptions.ConnectionError'> HTTPConnectionPool(host='127.0.0.1', port=62485): Max retries exceeded with url: /report (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x17e079290>: Failed to establish a new connection: [Errno 61] Connection refused')) [15:05:40.747, timer.py:36]
blenderkit INFO: BlenderKit-Client v0.0.3 starting on http://127.0.0.1:62485, log file at: /Users/ag/blenderkit_data/client/default.log [15:05:40.765, daemon_lib.py:538]
blenderkit INFO: BlenderKit-Client is running on port 62485! [15:05:40.856, timer.py:83]
blenderkit INFO: Unregistering BlenderKit add-on [15:05:41.478, __init__.py:2401]
blenderkit INFO: Reported Blender quit to Client. [15:05:41.485, __init__.py:2423]
blenderkit addon: Verbose is enabled
blenderkit updater: Running background check for update
blenderkit addon: Starting background checking thread
blenderkit addon: Checking for update now in background
blenderkit addon: Checking for update function
blenderkit INFO: Global dir permissions are OK. [15:05:41.735, utils.py:1275]
Writing: /var/folders/n5/zgtk48652gq_1_b8h9g2mfph0000gn/T/blender.crash.txt
[1]    22623 segmentation fault  /Applications/Blender.app/Contents/MacOS/blender
```

For Client it can happen more often (but not on first enable), hower for Daemon it can happen even on first enable.

I am disabling the Welcome popup window, as it is much better than having users to face crashing Blender.

Please take a look on implementation of queue worker, what could be wrong. I believe it is connected to the fake context or something like that.

Reply:
In all cases, the Welcome Window will not open in Blender 4.1. So disabling it only prevents the crashes, even when it does not crash, the popup Welcome is not shown. For that one has to disable fake context. But then crash will apear after it is closed.

Looks like queue worker needs some investigation and care really. @vilemduha